src/components/FieldNumber/FieldNumber.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import classNames from 'classnames';
+import { Field } from 'react-final-form';
+
+import css from './FieldNumber.module.css';
+
+const decrement = 'decrement';
+const increment = 'increment';
+
+const NumberInput = props => {
+  const { value: rawValue, onChange } = props.input;
+  const { initialValue, minValue = 0, maxValue } = props;
+  const value = rawValue
+    ? Number.parseInt(rawValue)
+    : initialValue
+    ? Number.parseInt(initialValue)
+    : minValue;
+
+  const handleValueChange = event => {
+    const { name } = event.target;
+    if (name === increment) {
+      onChange(value + 1);
+    } else {
+      onChange(value - 1);
+    }
+  };
+
+  console.log({ value });
+
+  return (
+    <span className={css.numberInputWrapper}>
+      <button
+        className={css.numberButton}
+        name={decrement}
+        type="button"
+        onClick={handleValueChange}
+        disabled={value <= minValue}
+      >
+        -
```

Comment:
Yes. The plus and minus should be SVG icons.
You would also need to center them, which you usually can do with flexbox (and sometimes you might need to use a padding for fine-tuning the vertical position).

> _Inside a button it might be somewhat tricky to position svg & text correctly (browsers have their own layout rules for buttons, which makes life sometimes difficult). Anyway, it does not look like you need to do that here, but an old trick has been to add <span> inside the button which then wraps all the content there. That way, you ppl can control the display mode and centering a bit better._

Reply:
Modified to SVGs

<img width="344" alt="Screenshot 2024-12-03 at 17 46 54" src="https://github.com/user-attachments/assets/74a33ea5-4fe4-4ab5-b88c-22adcec546c6">